### ISSUE TITLE
Define a default neural network architecture

### ISSUE THREAD

**Issue by williamjameshandley** [DarkMachines/pyBAMBI#16]
We need to try and define a default neural network architecture, loss function, training scheme and activation function, that works reasonably well on a variety of likelihood functions. This can be tested in parallel with #4 using an external script which @mjw63  can supply. It relies on samples from #15 being available.

Also involved: @melli1992
